packages/environment-check/src/checks/service-checks.ts

```diff
@@ -246,6 +246,16 @@ export async function checkTransportRequests(
         }
     } catch (e) {
         logger.error(t('error.getTransportRequestsError'));
+
```

Comment:
If you are handling the just the HTTP response, then it shoud be;
```
if (response.status === 403)) {
            logger.warn(t('warning.guidedAnswersLink'));
        }
```

Reply:
Thanks, changed to
```
if (e.response?.status === 403) {
            logger.warn(t('warning.guidedAnswersLink'));
}
```